packages/front/.env.example

```diff
@@ -1 +1,2 @@
-API_URL=http://localhost:3000
\ No newline at end of file
+API_URL=http://localhost:3000
+RECAPTCHA_SITEKEY=6Lc2BtYUAAAAAOUBI-9r1sDJUIfG2nt6C43noOXh
```

Comment:
@thesocialdev let's start adding newline at EOF on the next PRs

Reply:
Done.